pkg/util/kubernetes.go

```diff
@@ -467,3 +468,21 @@ func WaitUntilDeploymentConfigReady(c oc_cs.Interface, meta metav1.ObjectMeta) e
 		return false, nil
 	})
 }
+
+func WaitUntilVolumeSnapshotReady(c snapshot_cs.Interface, meta metav1.ObjectMeta) error {
+	return wait.PollImmediate(RetryInterval, 30*time.Minute, func() (bool, error) {
```

Comment:
I think we need a user configurable timeout. 30min is just abitrary.

Reply:
Agreed. But how can we do it? Any field in `BackupConfiguration`?

Reply:
We have increased the timeout to 2 hour